test/LinkCommand.js

```diff
@@ -0,0 +1,72 @@
+import log from "npmlog";
+
+// mocked or stubbed modules
+import FileSystemUtilities from "../src/FileSystemUtilities";
+
+// helpers
+import callsBack from "./helpers/callsBack";
+import exitWithCode from "./helpers/exitWithCode";
+import initFixture from "./helpers/initFixture";
+import normalizeRelativeDir from "./helpers/normalizeRelativeDir";
+
+// file under test
+import LinkCommand from "../src/commands/LinkCommand";
```

Comment:
The test needs to be updated to support the [yargsRunner pattern](https://github.com/lerna/lerna/commit/ab3c5c5cb42df9705e51eef970b1db263db9c813):

```js
import yargsRunner from "./helpers/yargsRunner";

// file under test
import * as commandModule from "../src/commands/LinkCommand";

const run = yargsRunner(commandModule);
```

The test case itself becomes much more compact:
```js
  describe("with local package dependencies", () => {
    beforeEach(stubSymlink);
    afterEach(resetSymlink);

    it("should symlink all packages", async () => {
      const testDir = await initFixture("LinkCommand/basic");
      const lernaLink = run(testDir);
      await lernaLink();

      expect(symlinkedDirectories(testDir)).toMatchSnapshot();
    });
  });
```

Thanks!